src/store/actions/keymap.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from '@angular/core';
+import { Action } from '@ngrx/store';
+
+import { Keymap } from '../../config-serializer/config-items/Keymap';
+
+@Injectable()
+export class KeymapActions {
+    static PREFIX = '[Keymap] ';
```

Comment:
How should I implement const into class static or did you have a different approach in mind?